app/build.gradle

```diff
@@ -15,9 +15,18 @@ def versionMajor = 0
 def versionMinor = 1
 def versionPatch = 0
 
+def getGitTimetamp() {
```

Comment:
A `s`is missing :)
Also I prefer this format, more readable: https://github.com/vector-im/riot-android/blob/develop/vector/build.gradle#L193

Reply:
> A sis missing :)

:see_no_evil: 

>  Also I prefer this format, more readable: https://github.com/vector-im/riot-android/blob/develop/vector/build.gradle#L193

What exactly do you mean here? The syntax for the gradle function? I can change that, sure.

Reply:
Yes, it was the format of the gradle function, thanks for the change